cmake/Intel.cmake

```diff
@@ -1,5 +1,6 @@
 set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -traceback -fpp -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -sox -align array64byte -qno-opt-dynamic-align")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -qno-opt-dynamic-align -sox -fp-model source")
+#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -qno-opt-dynamic-align -sox -fp-model source")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -qno-opt-dynamic-align -fp-model precise")
```

Comment:
We shouldn' commit this to the repo. Pleae have a look at https://github.com/ufs-community/ufs-weather-model/pull/2504 for how this should be done.

Reply:
@climbfuji totally agree -- was mostly committing here so i did not forget. wanted to sync up w/ you/dusan/jong/ratko about how to sync up the 1.9.1 WM PR (moving to mixed compilers), your PR here, and the ursa PR, as ratko/my PRs will need your changes.

Reply:
Good luck staying on top of all these moving pieces ;-) but sounds good